trade-finance-manager-ui/server/controllers/case/amendments/amendmentFacilityEndDate.post.api-test.js

```diff
@@ -219,7 +221,7 @@ describe('amendmentFacilityEndDate routes', () => {
 
     describe('correct facility end date entered', () => {
       it('should redirect to the check answers page when only the cover end date is being amended and there are no errors', async () => {
-        const mockFacility = { facilitySnapshot: { dates: { coverStartDate: new Date(2024, 1, 1).valueOf().toString() } } };
+        const mockFacility = { facilitySnapshot: { type: gefFacilityType, dates: { coverStartDate: new Date(2024, 1, 1).valueOf().toString() } } };
```

Comment:
can `new Date(2024, 1, 1).valueOf().toString()` be declared as a constant on top? cleaner to look at and has been repeated